pwiz_tools/Skyline/Controls/Graphs/AreaReplicateGraphPane.cs

```diff
@@ -489,7 +495,11 @@ public override void UpdateGraph(bool selectionChanged)
                     }
 
                     // Add area for this transition to each area entry
-                    AddAreasToSums(pointPairList, sumAreas);
+                    Func<double,double, double> aggregateFunc = (sums, y) => sums + y;
+                    if (BarSettings.Type == BarType.Cluster)

```

Comment:
I do not understand what this code here is for. If the bar is a cluster then we want to take the Max of all the values instead of summing them? Sounds plausible, but maybe there should be a comment saying why we're doing this.

Reply:
Yes, that is correct. For a cluster graph we need to use max instead of sum.